engine/access/rest/websockets/models/event_models.go

```diff
@@ -0,0 +1,16 @@
+package models
+
+import (
+	"time"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// EventResponse is the response message for 'events' topic.
+type EventResponse struct {
+	BlockId        string       `json:"block_id"`
+	BlockHeight    string       `json:"block_height"`
+	BlockTimestamp time.Time    `json:"block_timestamp"`
+	Events         []flow.Event `json:"events"`
+	MessageIndex   string       `json:"message_index"`
```

Comment:
I think it would be much easier for the user if message id came as an integer value. are you using string here because it's a uint64?